spec/factories/conference_attendances.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :conference_attendance do
+    attendance false
+    orga_comment "MyText"
```

Comment:
Probably mostly personal preference, but I found it easier on the long run, to keep factories as simple as possible. Meaning, if the `orga_comment` is an optional attribute, I'd just fully skip it in the factory.